adding details about service user (#1291)
dbt-labs/docs.getdbt.com: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/setup-azure.md

> 3. [Connect Azure DevOps to your new app](#connect-azure-devops-to-your-new-app).

Reviewer: ```suggestion
3. [Add another redirect URI](#add-another-redirect-URI).
4. [Connect Azure DevOps to your new app](#connect-azure-devops-to-your-new-app).
```

> 4. [Add your Azure AD app to dbt Cloud](#add-your-azure-ad-app-to-dbt-cloud).

Reviewer: ```suggestion
5. [Add your Azure AD app to dbt Cloud](#add-your-azure-ad-app-to-dbt-cloud).
```

Provide your new app access to Azure DevOps:

1. Select **API permissions** in the left navigation panel.
2. Remove the **Microsoft Graph / User Read** permission.
3. Click **Add a permission**.
4. Select **Azure DevOps**.
5. Select the **user_impersonation** permission.

<Lightbox src="/img/docs/dbt-cloud/connecting-azure-devops/user-impersonation.gif" title="Adding permissions to the app"/>

> You will also need to add an additional redirect URI to your Azure AD application which will be used to authenticate the service user for headless actions in deployment environments.

Reviewer: ```suggestion
## Add another redirect URI 

You also need to add an another redirect URI to your Azure AD application. This redirect URI will be used to authenticate the service user for headless actions in deployment environments.
```

1. Navigate to your Azure AD application.
2. Select the link next to **Redirect URIs**

> 3. Add an additional URI `https://cloud.getdbt.com/complete/azure_active_directory_service_user`. If you have a custom dbt Cloud URL be sure to use the appropriate domain.

Reviewer: ```suggestion
3. Click **Add URI** and add the URI, making sure to use the appropriate domain if you have a custom dbt Cloud URL:
`https://cloud.getdbt.com/complete/azure_active_directory_service_user`
```

Your Azure AD app should now be added to your dbt Cloud Account. People on your team who want to develop in dbt Cloud's IDE can now personally [authorize Azure DevOps from their profiles](dbt-cloud/cloud-configuring-dbt-cloud/authenticate-azure).

## Connecting a service user

> Azure DevOps' permissions are tightly coupled to a developer's identity. However, in deployment environments in dbt Cloud, we don't want runs to be tied to an individual's identity as that would create production problems should an individual lose access to a dbt project repository. We encourage our customers to create service account users in Azure DevOps for the purposes of powering headless actions in dbt Cloud deployment environments. This service user must have read access to all dbt repos across projects in the dbt Cloud account. dbt Cloud will refresh the OAuth access token regularly.

Reviewer: ```suggestion
Azure DevOps' permissions are tightly coupled to a developer's identity. However, in dbt Cloud deployment environments, you should avoid tying runs to an individual's identity as that creates production problems if an individual loses access to a dbt project repository. 

Instead, we recommend creating service account users in Azure DevOps to power headless actions in dbt Cloud deployment environments. This service user must have read access to all dbt repos across projects in the dbt Cloud account. dbt Cloud will refresh the OAuth access token regularly.
```

> This service user's permissions will also power which repos a team can select from during dbt project set up, so an Azure DevOps admin must grant read access to the service user before setting up a project in dbt Cloud.

Reviewer: ```suggestion
:::info  Azure DevOps admin must grant read access to the service user
This service user's permissions will also power which repos a team can select from during dbt project set up, so an Azure DevOps admin must grant read access to the service user before setting up a project in dbt Cloud.
:::
```

To connect the service user:
1. An admin must first be signed into the service user's Azure DevOps account.
2. The admin should click **Link Azure Service User** in dbt Cloud.
3. The admin will be directed to Azure DevOps and must accept the Azure AD app's permissions.
4. Finally, the admin will be redirected to dbt Cloud, and the service user will be connected.
<Lightbox src="/img/docs/dbt-cloud/connecting-azure-devops/azure-service-user.png" title="Connecting an Azure Service User"/>

> Once connected, dbt Cloud will display the email address of the service user so as to explicitly show which user's permissions are enabling headless actions in deployment environments. To change which account is connected, disconnect the profile in dbt Cloud, sign into the alternative Azure DevOps service account, and re-link the account in dbt Cloud.

Reviewer: ```suggestion
Once connected, dbt Cloud displays the email address of the service user so you know which user's permissions are enabling headless actions in deployment environments. To change which account is connected, disconnect the profile in dbt Cloud, sign into the alternative Azure DevOps service account, and re-link the account in dbt Cloud.
```
